### PR TITLE
[AKS]Leave only one default role for Automatic clusters

### PR DESCRIPTION
**Pull request by purpleran7**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/aks-preview/HISTORY.rst [hunk 1]
```diff
@@ -13,6 +13,11 @@ Pending
 +++++++
 * Improve Windows OutboundNat test case by removing Windows OSSKU limitation
 
+4.0.0b2
+++++++++
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:17]:
```suggestion

4.0.0b2
+++++++
* Improve Windows OutboundNat test case by removing Windows OSSKU limitation
```

**Review by FumingZhang — COMMENTED** [review]
Please queue a live test to validate the change.

**Comment by purpleran7** [comment]
Live Test : https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93531457&view=results

**Comment by purpleran7** [comment]
Live test passed today
 https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93602522&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b.

**Pull request closed without merging**